src/Apps/Notifications/notificationsRoutes.ts

```diff
@@ -0,0 +1,20 @@
+import loadable from "@loadable/component"
+import { AppRouteConfig } from "System/Router/Route"
+
+const NotificationsApp = loadable(
+  () =>
+    import(/* webpackChunkName: "notificationsBundle" */ "./NotificationsApp"),
+  {
+    resolveComponent: component => component.NotificationsApp,
+  }
+)
+
+export const notificationsRoutes: AppRouteConfig[] = [
+  {
+    path: "/notifications",
```

Comment:
**question**: do you think such a path is suitable or is it better to use another one (for example, `/activities`)?